mobile/status.go

```diff
@@ -1088,3 +1088,9 @@ func ToChecksumAddress(address string) string {
 	}
 	return address
 }
+
+func ConvertDesktopKeyToMobileKey(DesktopKey string) string {
```

Comment:
I am not sure if we should stick with the concepts of "desktop key" or "mobile key". In my understanding, both platforms are currently using different key formats but in the end all clients should use the same one, so if this is true, the function name could create confusion. Maybe using a more generic naming as `DeserializeAndCompressKey` is more accurate, which in the end is what this function is doing, so is self-descriptive. What do you think?

Reply:
Hi @briansztamfater sounds good I will rename this to `DeserializeAndCompressKey`